### DISCUSSION TITLE
Multiple candidates with Collab?

### DISCUSSION THREAD

**Discussion by protomato2** [neonbjb/tortoise-tts#225] [General]
First of all, thank you for this amazing program!
I noticed it's much easier to achieve a necessary result if there are 3 output results as compared to just one. However my GPU - 1050 Ti - is pretty weak and I'd rather use the Collab notebook. But Collab notebook only produces singular output. Is there a way to make it produce 3 samples like on local setup? I lack any programming knowledge and I can't think of anything that would work.

**Comment by protomato2** [comment]
Hmm, it says there is 1 comment, yet there are no comments

**Comment by neonbjb** [comment]
Hey - try looking up how to run commands in commands, then follow those instructions in collab. Then run the commands in https://github.com/neonbjb/tortoise-tts#readpy

**Reply by protomato2** [reply]
> Hey - try looking up how to run commands in commands, then follow those instructions in collab. Then run the commands in https://github.com/neonbjb/tortoise-tts#readpy

Thank you for the response!
So, I just run the command for local install in Collab? I will try
